### PR TITLE
Update PVC resizing documentation

### PR DESCRIPTION
**Pull request by gnufied**
Xref https://github.com/kubernetes/features/issues/284

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6487)

### CODE REVIEW

File: docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -118,8 +118,12 @@ For volume plugins that support the Delete reclaim policy, deletion removes both
 
 ### Expanding Persistent Volumes Claims
 
-With Kubernetes 1.8, we have added Alpha support for expanding persistent volumes. The current Alpha support was designed to only support volume types
-that don't need file system resizing (Currently only glusterfs).
+With Kubernetes 1.8, we have added Alpha support for expanding persistent volumes. We currently following volume types:
```
Comment (tengqm) [inline docs/concepts/storage/persistent-volumes.md]: >We currently following volume types:

In v1.9, the following types have volume expansion support:

File: docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -118,8 +118,12 @@ For volume plugins that support the Delete reclaim policy, deletion removes both
...
+
+* GCEPersistentDisk
+* AWSElasticBlockStore
+* CinderVolumeSource
+* GlusterfsVolumeSource
```
Comment (tengqm) [inline docs/concepts/storage/persistent-volumes.md]: * `awsElasticBlockStore`
* `cinder`
* `gcePersistentDisk`
* `glusterfs`
* `rbd`

File: docs/concepts/storage/persistent-volumes.md (hunk 1 above)
Comment (gnufied) [reply docs/concepts/storage/persistent-volumes.md]: fixed

File: docs/concepts/storage/persistent-volumes.md (hunk 2 above)
Comment (gnufied) [reply docs/concepts/storage/persistent-volumes.md]: fixed

File: docs/concepts/storage/persistent-volumes.md [hunk 3]
```diff
@@ -118,8 +118,14 @@ For volume plugins that support the Delete reclaim policy, deletion removes both
 
 ### Expanding Persistent Volumes Claims
 
-With Kubernetes 1.8, we have added Alpha support for expanding persistent volumes. The current Alpha support was designed to only support volume types
-that don't need file system resizing (Currently only glusterfs).
+With Kubernetes 1.8, we have added Alpha support for expanding persistent volumes. In v1.9, the following volume
```
Comment (zacharysarah) [inline docs/concepts/storage/persistent-volumes.md]: Merge this line and the one following:
```
Kubernetes 1.8 added Alpha support for expanding persistent volumes. In v1.9, the following volume types support expanding Persistent volume claims:
```
So:
> Kubernetes 1.8 added Alpha support for expanding persistent volumes. In v1.9, the following volume types support expanding Persistent volume claims:

File: docs/concepts/storage/persistent-volumes.md [hunk 4]
```diff
@@ -147,6 +153,16 @@ by simply editing the claim and requesting bigger size.  This in turn will trigg
 
 Under no circumstances a new `PersistentVolume` gets created to satisfy the claim. Kubernetes will attempt to resize existing volume to satisfy the claim.
 
+If volume being expanded contains a File System then file system resizing is only performed when a new Pod is started using the `PersistentVolumeClaim` in
```
Comment (zacharysarah) [inline docs/concepts/storage/persistent-volumes.md]:
```
For expanding volumes containing a file system, file system resizing is only performed when a new Pod is started using the `PersistentVolumeClaim` in
```

File: docs/concepts/storage/persistent-volumes.md [hunk 5]
```diff
@@ -147,6 +153,16 @@ by simply editing the claim and requesting bigger size.  This in turn will trigg
...
+ReadWrite mode. In other words, if volume being expanded is used in a pod or deployment - you will need to delete and recreate the pod for file system
```
Comment (zacharysarah) [inline docs/concepts/storage/persistent-volumes.md]:
```
ReadWrite mode. In other words, if a volume being expanded is used in a pod or deployment, you will need to delete and recreate the pod for file system
```

File: docs/concepts/storage/persistent-volumes.md [hunk 6]
```diff
@@ -147,6 +153,16 @@ by simply editing the claim and requesting bigger size.  This in turn will trigg
...
+resizing to take place. Also file system resizing is only supported for following file system types:
```
Comment (zacharysarah) [inline docs/concepts/storage/persistent-volumes.md]:
```
resizing to take place. Also, file system resizing is only supported for following file system types:
```

File: docs/concepts/storage/persistent-volumes.md [hunk 7]
```diff
@@ -147,6 +153,16 @@ by simply editing the claim and requesting bigger size.  This in turn will trigg
...
+
+* XFS
+* Ext3, Ext4
+
+#### A note about expanding EBS volumes
```
Comment (zacharysarah) [inline docs/concepts/storage/persistent-volumes.md]: Replace this heading and the following two lines with:
```
**Note:** Expanding EBS volumes is a time consuming operation. Also, there is a per-volume quota of one modification every 6 hours.
{: .note}
```

**Comment by gnufied** [comment]
@zacharysarah addressed review comments. PTAL

File: docs/concepts/storage/persistent-volumes.md [hunk 8]
```diff
@@ -118,8 +118,13 @@ For volume plugins that support the Delete reclaim policy, deletion removes both
 
 ### Expanding Persistent Volumes Claims
 
-With Kubernetes 1.8, we have added Alpha support for expanding persistent volumes. The current Alpha support was designed to only support volume types
-that don't need file system resizing (Currently only glusterfs).
+Kubernetes 1.8 added Alpha support for expanding persistent volumes. In v1.9, the following volume types support expanding Persistent volume claims:
+
+* gcePersistentDisk
+* awsElasticBlockStore
+* Cinder
```
Comment (tengqm) [inline docs/concepts/storage/persistent-volumes.md]: nit: Cinder should be cinder

**Pull request merged**